DOCUMENTATION.md

```diff
@@ -401,6 +401,11 @@ File.open("path/to/pdf_file", "rb") do |pdf_file|
 end
 ```
 
+#### postage (optional)
+
+You can select first or second class postage for your precompiled letter. Set the value to `first` for first class, or `second` for second class. If you do not pass in this argument, the postage will default to second class.
+
```

Comment:
Thanks for making the changes @CrystalPea, is this just for the Ruby client? Or does it apply to multiple / all clients?

Reply:
Hi @jonathanglassman , no worries, it will be all the clients, and some small things in the code (adding argument to request and to expected response) have to be changed alongside the documentation in each of them.

Reply:
Minor thing: 'choose' vs 'select'

What do you think @jonathanglassman?

Reply:
Either works. Choose is plainer english so that is probably better :-)